styles/styles.css

```diff
@@ -1,124 +1,291 @@
+    margin: 0;
 }
-h2 {
-   color: #1f4796;
+
+/*Lesson 5*/
+/*WHAT DO WE OFFER*/
+
+.cont {
+    max-width: 1440px;
+    padding: 100px 16px 0;
+    margin: 0 auto;
 }
-h3 {
-    color: #4d8d4d;
-    margin-bottom: 30px;
+
+.cont_title {
+    font-size: 40px;
+    font-weight: 500;
+    text-align: center;
+    margin: 120px auto 20px;
 }
-h4 {
-    margin-bottom: 30px;
+
+.cont_line {
+    border: none;
+    background: #F5BD41;
+    overflow: visible;
+    width: 100px;
+    height: 4px;
+    margin: 70px auto 100px;
 }
-h5 {
-    margin-bottom: 30px;
+
+.row {
+    display: flex;
+    gap: 16px;
+    margin: 20px 104px 115px;
 }
-.img {
-    max-width: 350px;
-    min-height: 450px;
+
+.col {
+    width: 25%;
 }
-table, td, th {
-     border: 1px solid black;
- }
-table {
-    border-collapse: collapse;
-    width: 80%;
+
+.cont_row {
+    margin-bottom: 120px;
 }
-th {
-    height: 50px;
-    text-align: left;
+
+.cont_col {
+    text-align: center;
 }
-tr:hover {
-    background-color: #deba97;
+
+.cont_icon {
+    max-width: 100px;
+    min-height: 100px;
+    width: auto;
 }
-a:hover {
-    color: #3b3a3a;
+
+.cont_title_icon {
+    font-size: 24px;
+    line-height: 28px;
+    margin: 65px auto 10px;
+    font-weight: 400;
+    text-align: center;
+    color: #383838;
 }
-input[type="text"]
-{
-    color: #e79898;
+
+/*FOOTER*/
+
+.flex-cont-footer {
+    background: #10367C;
+    max-width: 1440px;
+    height: 630px;
+    margin: 0;
+    border: 1px solid #000000;
+    box-shadow: 0 4px 4px;
+}
+
+.cont_block_row {
+    display: flex;
 }
-ol::before {
-    content: " ! The first step towards the development !";
-    background-color: #d7d77b;
-    color: #ad3c3c;
-    font-weight: bold;
+
+.cont_col_about {
+    width: 193px;
+    height: 211px;
+    margin: 120px 222px 0 104px;
+    color: #ffffff;
 }
-ol::after {
-    content: " ! The first step towards the development has been taken !";
-    background-color: #d7d77b;
-    color: #ad3c3c;
-    font-weight: bold;
+
+.cont_col_property {
+    width: 134px;
+    height: 310px;
+    margin: 120px 282px 0 0;
+    color: #ffffff;
 }
-.input {
-    min-inline-size: auto;
-    /*height: 100px;*/
+
+.cont_col_support {
+    width: 233px;
+    height: 122px;
+    margin: 120px 272px 0 0;
+    color: #ffffff;
 }
-.button {
-    min-width: 70px;
+
+.row_1.row_2.row_3 {
+    line-height: 23px;
 }
-.form_fieldset {
-    margin-bottom: 30px;
+
+#footer {
+   margin: 104px 0 1080px 120px;
+    color: #ffffff;
 }
-/*Lesson 4 */
-.advantage {
-    max-width: 1480px;
+
+/* CONTAINER */
+.page__container {
+    width: 100%;
+    max-width: 1440px;
     margin: 0 auto;
 }
-.advantage__title {
-    text-align: center;
-    font-style: normal;
-    font-weight: 500;
-    font-size: 40px;
-    line-height: 90%;
-    color: #333333;
-    margin-bottom: 50px;
+
+/* Header */
+.header {
+    width: 100%;
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    z-index: 1000;
```

Comment:
Oh no, only without `position: absolute`
You can add a `div` wrapper and include the `header` and `form block` in it and add a background image for that `div` wrapper. Then you can delete `position: absolute` and everything connected with it